### ISSUE TITLE
Ionic tabs with nested states navigation issue

### ISSUE THREAD

**Issue by roocster** [driftyco/ionic#6547]
When navigate on the tab, the last visited nested view is displayed on it.
http://codepen.io/roocster/pen/pymeNL?editors=1010

**Steps to reproduce:**
1. Go to "Tab 2"
2. Go to "Substate 1"
3. Return to "Tab 1"
4. Go to "Tab 2"
Instead of the state "tabs.tab-2.main" ("**Tab 2. Main**" in view title) it goes to the previous visited state "tabs.tab-2.substate-1" ("**Tab 2. Substate 1**" in view title). But the second tab is linked to "tabs.tab-2.main" state by ui-sref directive:
`<ion-tab ui-sref="tabs.tab-2.main" title="Tab 2">`

The necessary state "tabs.tab-2.main" is loaded only after double clicking on the "Tab 2".

**Comment by speciose** [comment]
> The necessary state "tabs.tab-2.main" is loaded only after double clicking on the "Tab 2".

Have similar issue with NavController. Mostly it works after second click only

**Comment by jgw96** [comment]
Hey @roocster ! Thanks for opening an issue with us! I am going to close this as a duplicate of https://github.com/driftyco/ionic/issues/6527, would you mind commenting about your issue on that issue please? Thanks for using Ionic!
